spec-draft.md

```diff
@@ -272,10 +272,65 @@ as follows:
 * The virtual keyboard: use sendKeys
 * acceleromator: TODO @mdas is working on this
 * geolocation: use regular webdriver endpoints
-* battery level: not in spec, perhaps exposed via executeScript
 * network speed: not in spec, perhaps exposed via executeScript
 
 
+Battery
+-------
+Device battery changes might trigger events on users applications, controlling this will help monitor and verify how apps responds to these factors.
+
+The battery state should have two options: ON, OFF depending if the power ac state it's charging or not.
+
+* GET /session/:sessionId/appium/device/battery_state
```

Comment:
here and in all the following, we don't need `appium/device` in the endpoint. this spec draft is a draft of what would potentially be an official spec at some point. the `appium/` prefix is for appium-specific functionality that would not be a part of this draft. or if we wanted to implement this in appium before its admission into the draft.

so i would recommend one of these two:

```
GET /session/:sessionId/device/battery_state
GET /session/:sessionId/battery_state
```

if we have enough things to put behind `/device` that could be a reasonable global prefix